doc/news/changes/minor/20230707BrunoBlais

```diff
@@ -0,0 +1,3 @@
+Improvement: Step-68 now uses the FEPointEvaluation to calculate the particle velocity from the velocity solution instead of manually interpolating at the particle location. 
+<br>
+(Bruno Blais, 2023/07/0)
```

Comment:
Do you want to set the right date?
```suggestion
(Bruno Blais, 2023/07/07)
```
Also, please break the first line at 80 characters.